e2eAIOK/DeNas/pruner/pruner.py

```diff
@@ -0,0 +1,21 @@
+import torch.nn.utils.prune as prune
```

Comment:
Is this file expected to be the pruner algorithm file? make it pluggable to support other algorithms. 

So only two were supported below, how the two algos were chosen?

Reply:
the pruning algos can be configed in Pruner config file with `algo` config